### DISCUSSION TITLE
Question about porting and slight confusion?

### DISCUSSION THREAD

**Discussion by Pinkyyyy** [NeoArmageddon/co10_Escape#106] [Q&A]
Hi, so nooby question here. I would like to learn how to port escape to other terrains since my group has been pretty good at escaping. I am however a bit confused, in the editing and porting folder there is a porting folder and a raw mission folder. The porting folder links to an oldish thread about how to port to different terrains while the raw mission folder has a mission.sqm in it and the description reads " Added base mission.sqm as starting point for editing" so..im a bit confused what the..mission.sqm is used for here....is there now a new way of porting escape or should i still stick to the forum post?

Would highly appreciate any links to videos or maybe even stream vods of people porting terrains, or any other guides. Im mostly asking this because i dont want to follow the forum post and then halfway through find out theres a better way of doing this.
Also excuse me for such a nooby question, im very new to this, and github. I only ported a DRO mission once (and it was a bit broken but hey it mostly worked haha)

Anyhow thanks if you read through this mess, and again apologize for such a nooby question.

**Comment by CaptainPStar** [comment]
_ i dont want to follow the forum post and then halfway through find out theres a better way of doing this._
Yes, that would definitly be the case by just reading the BIF thread in a chronological order.

The porting info is a bit  outdated because we are currently working a lot on the underlying systems to make it easier to port and add new base templates and stuff. I will try to collect some up-to-date info on porting till weekend (or rather make @ScruffyAT write it down).

In the mean time you can ask questions in our official Discord server https://discord.gg/pCBfKVrc
Some people there are quite experienced with porting escape.

**Reply by Pinkyyyy** [reply]
Thanks! I saw something about new templates in the issues, but didnt really understand it.
